screwdriver.yaml

```diff
@@ -20,14 +20,12 @@ jobs:
             - setup-ci: git clone https://gist.github.com/3d2388b2a7ba658cdcdaffa8cd874e50.git ci
             - get: go get -t ./...
             - gofmt: "find . -name '*.go' | xargs gofmt -s -w"
-            - build: go build -a -o sd-step
-            - get-bzip2: apt-get update && apt-get install -y --no-install-recommends bzip2
             - tag: ./ci/git-tag.sh
-            - release: ./ci/git-release.sh
+            - release: |
+                rm -f ./VERSION
```

Comment:
I'm not sure why you're removing the `VERSION` file. You should be able to modify the files being uploaded in the `.goreleaser.yml` file:

* https://github.com/goreleaser/goreleaser/blob/master/README.md#archive-customization

```
  # Additional files/globs you want to add to the archive.
  # Defaults are any files matching `LICENCE*`, `LICENSE*`,
  # `README*` and `CHANGELOG*` (case-insensitive)
```